docs/DeveloperGuide.md

```diff
@@ -624,202 +625,210 @@ testers are expected to do more *exploratory* testing.
 
 ### Launch and shutdown
 
-1. Initial launch
+1. Launching the app
+   1. Prerequisites: Have a copy of `Trackermon.jar` in your computer.
+      1. [Download](https://github.com/AY2122S2-CS2103T-T09-3/tp/releases) the jar file and copy the file into an empty folder.
+   2. Double-click the jar file.
+   3. Test case: Initial launch. <br> 
+      Expected: Application launched with a set of sample shows.
+   4. Test case: Subsequent Launch. <br> 
+      Expected: Application launched with user saved shows.
+
+2. Saving window preferences
+   1. Launch the app. <br> 
+      Expected: The window size may not be optimum.
+   2. Resize the window to an optimum size. Move the window to a different location. Close the window.
+   3. Re-launch the app. <br> 
+      Expected: The most recent window size and location is retained.
 
-    1. Download the jar file and copy into an empty folder
-
-    1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
-
-1. Saving window preferences
+[return to top <img src="images/toc-icon.png" width="25px">](#table-of-contents)
 
-    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
+---
 
-    1. Re-launch the app by double-clicking the jar file.<br>
-       Expected: The most recent window size and location is retained.
+### Adding a show
 
-1. _{ more test cases …​ }_
+1. Prerequisites: None.
+2. Test case: Adding a valid show
+   1. Condition: Show named `Inception` must not exist in the show list.
+   2. Command: `add n/Inception s/watching` <br> 
+      Expected: Show is added into the show list. Added show details shown in the result display.
+3. Test case: Adding an invalid show
+   1. Condition: Show named `"Inception"` exists in the show list.
+   2. Command: `add n/Inception s/watching` <br> 
+      Expected: Show is not added into the show list. Error details shown in the result display.
 
 [return to top <img src="images/toc-icon.png" width="25px">](#table-of-contents)
 
 ---
 
 ### Deleting a show
 
-1. Deleting a show while all shows are being shown
+1. Prerequisites: None, but if the list is empty, all deletions will result in an error.
+   1. [Add shows](#adding-a-show) into the show list.
+2. List all shows using the `list` command.
+3. Test case: Deleting a show based on list index
+   1. Condition: Range of show must be within range of size of show list.
+   2. Command: `delete 1` <br> 
+      Expected: Show at specified index is deleted from the list. Deleted show details shown in the result display.
+4. Test case: Invalid delete index
+   1. Condition: Range of show must be outside of list size.
```

Comment:
Should this be `Range of shows` instead.

Reply:
Edited